beta/index.markdown

```diff
@@ -0,0 +1,183 @@
+---
```

Comment:
Let's make a linter test that fails on index.anything and suggests this. I dislike simple policies that humans have to enforce in reviews.